oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -79,12 +80,16 @@ public class LinuxOperatingSystem extends AbstractOperatingSystem {
         boolean hasGettid = false;
         boolean hasSyscallGettid = false;
         try {
-            try {
-                @SuppressWarnings("unused")
-                Udev lib = Udev.INSTANCE;
-                hasUdev = true;
-            } catch (UnsatisfiedLinkError e) {
-                LOG.warn("Did not find udev library in operating system. Some features may not work.");
+            if (GlobalConfig.get(GlobalConfig.OSHI_OS_LINUX_ALLOWUDEV, true)) {
+                try {
+                    @SuppressWarnings("unused")
+                    Udev lib = Udev.INSTANCE;
+                    hasUdev = true;
+                } catch (UnsatisfiedLinkError e) {
+                    LOG.warn("Did not find udev library in operating system. Some features may not work.");
+                }
+            } else {
+                LOG.warn("Loading of udev not allowed by configuration. Some features may not work.");
```

Comment:
"disallowed" maybe better? or bike-shedding?

Reply:
Agree with INFO and "Loading of udev disabled by configuration."

Reply:
Actually given the name, "not allowed" is fine.  Changed it to info.  Will merge when CI passes and cut a release before I go on vacation :)